modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/UpNextQueueImpl.kt

```diff
@@ -206,7 +206,7 @@ class UpNextQueueImpl @Inject constructor(
         val mutableList = episodes.toMutableList()
         if (isEmpty) {
             val firstEpisode = mutableList.first()
-            playNextNow(firstEpisode, downloadManager, null)
```

Comment:
There isn't a `playNextNow` after the methods were renamed to have a suffix `Blocking`, I think this is an issue I introduced into the `task/android-15` branch.